app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecord.kt

```diff
@@ -34,6 +34,19 @@ data class TeleconsultRecord(
     val syncStatus: SyncStatus
 ) {
 
+  fun toPayload(): TeleconsultRecordPayload {
```

Comment:
How can we get the `teleconsultRecord` object in the `TeleconsultRecordSync` class?
Ps. We've consistently converted the object into network representation class in the domain class across the project but you have a fair point as well.

Reply:
We used to do that initially but we decided to move this conversion to the sync classes later on. I don't think we ended up moving all the existing resources, sadly.

> How can we get the teleconsultRecord object in the TeleconsultRecordSync class?

You already have the instance since `TeleconsultRecordSync` calls `.toPayload()` on it, right? What I mean is that we should instead have a `recordToPayload(TeleconsultRecordSync): TeleconsultRecordPayload` method in `TeleconsultRecordSync` itself.